intercom_flutter/ios/Classes/IntercomFlutterPlugin.m

```diff
@@ -226,12 +226,33 @@ - (ICMUserAttributes *) getAttributes:(FlutterMethodCall *)call {
     if(userId != (id)[NSNull null]) {
         attributes.userId = userId;
     }
-    NSString *companyName = call.arguments[@"company"];
-    NSString *companyId = call.arguments[@"companyId"];
-    if(companyName != (id)[NSNull null] && companyId != (id)[NSNull null]) {
+    NSDictionary *companyDictionary = call.arguments[@"company"];
+    if(companyDictionary != (id)[NSNull null]) {
         ICMCompany *company = [ICMCompany new];
-        company.name = companyName;
-        company.companyId = companyId;
+        NSString *companyId = companyDictionary[@"id"];
+        if(companyId != (id)[NSNull null]) {
+            company.companyId = companyId;
+        }
+        NSString *name = companyDictionary[@"name"];
+        if(name != (id)[NSNull null]) {
+            company.name = name;
+        }
+        NSDate *createdAt = companyDictionary[@"createdAt"];
+        if(createdAt != (id)[NSNull null]) {
+            company.createdAt = createdAt;
```

Comment:
It should be a timestamp. Not sure about Objective-C, but it looks like it should be `NSNumber`

Reply:
Below is the part of interface ICMCompany:
`@interface ICMCompany : NSObject

/**
 The created at date for this company.
 */
@property (nonatomic, strong, nullable) NSDate *createdAt;`

In accordance with the above, and considering that in Flutter we use the `DateTime` type for the `createdAt` property, it seems to me that it should still be NSDate. Do you agree?

Reply:
@mozhaiskyi I got it. Since we are using `'createdAt': company.createdAt?.millisecondsSinceEpoch` we will probably need to convert that `int` into `NSDate`. Will fix this in the next commit.